docs/JOSS/paper.bib

```diff
@@ -0,0 +1,134 @@
+@techreport{Cranmer:1456844,
+      author        = "Cranmer, Kyle and Lewis, George and Moneta, Lorenzo and
+                       Shibata, Akira and Verkerke, Wouter",
+      title         = "{HistFactory: A tool for creating statistical models for
+                       use with RooFit and RooStats}",
+      institution   = "New York U.",
+      collaboration = "ROOT Collaboration",
```

Comment:
@cranmer should this be "ROOT collaboration"?

Reply:
FWIW, that's just copy pasted from the CDS entry's BibTeX https://cds.cern.ch/record/1456844/export/hx?ln=en.